pkg/k8s/apis/cilium.io/v2/client/register.go

```diff
@@ -124,7 +142,7 @@ func createCNPCRD(clientset apiextensionsclient.Interface) error {
 
 	res := &apiextensionsv1beta1.CustomResourceDefinition{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: k8sconstv2.CNPName,
+			Name: ciliumCRD.Spec.Names.Plural + "." + k8sconstv2.CustomResourceDefinitionGroup,
```

Comment:
Can you move this to a separate commit? Also, why does need to change?

Reply:
This is likely not needed. See [1]